tests/FormTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Modelizer\Selenium\Test;
+
+class FormTest extends TestCase
+{
+
+
+    public function testItCanType()
+    {
+
```

Comment:
Extra line. Please also check for other methods and after class started.
Also, I notice there are few PSR-2 fixes, which we can handle by StyleCI but it's a good practice to write code in PSR-2. :)